### PR TITLE
Add CryptPad to Paste Services

### PR DESCRIPTION
**Pull request by kryptish**
Adds CryptPad to paste services section.

### CODE REVIEW

**Comment by nitrohorse** [comment]
Hey @kryptish, thanks for submitting the PR! We usually look for an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues) as a pre-requisite; would you be able to create a “software suggestion” Issue to capture any discussion related to CryptPad?

**Comment by kryptish** [comment]
@nitrohorse done!

File: _includes/sections/paste-services.html [hunk 1]
```diff
@@ -7,4 +7,14 @@ <h1 id="paste" class="anchor"><a href="#paste"><i class="fas fa-link anchor-icon
...
...Celn1kAwBgpbW8wG0DsGdqu7z7KGABQIdGj7ulEu474C4cIJpRxTxj1A0BNqNSqxNi30i8fQvLFb1+r1EbSzyIAQEBs3F1l3GfplxHh8cWv3adE2zvSzx4AQBgFhOoTyriXcdOek37mAAABibW9lWi3If2SIibw4tfuA1v7AAD7SpZWf2CRYLVCGZtful+ckn62AAAlcCV1V5V2b6RfXsSxXvws8gMAHMHCylSi7R2l3SfplxkxQmi3EWt7S/rxAQCUXP9cAftM/MVGHObl36F+PwBgrBLdu0btgFDDvr2SuqvSzwgAoKoWVqZi4+7yWSCY2GRPPwDAm8Fxw8sBvADrG9p15h73zko/CwCAGuKzgP9gdT8AIAyD3QKJcZvSL8cqR7+YT3Fb+nYDAPCFuWbvDEWEJvTyN+5J9Kh7WvoeAwCwL5XaiLMFxvbifxnrVxek7ykAAIcWa3cz0XZd+iVaxlDavktSe136HgIAcCSzSy9OKG0fcuTwYV/87lOSFveixc609L0DAODY2DZ4iNCuw1G9AIBKYn3AXi9+u3o5LWal7w0AABNH7QC3lRj7Nk6Leel7AQCAN0oXt+VfwJKjfteRvgcAAHg31+ydEX8JC4Uy7vPs0osT0vcAAAARyriX0i9jkQRA2+fSbQ8AgBiVWiX9MhZJAKjjDwCou7otBlTadqXbHAAAcSq1kfRL2Wew3Q8AgAGl7XPpF7On0T/f/gEAGIoedU9XvVSwMu4zp/kBALBLrO0t6Zf0JCPW9pZ0GwMAEKRE21XpF/VEQttV6bYFACBYFx/Yk5U7PljbdYr+AADwDZHpzijtPom/uMcTm5Hpzki3KQAApXA5LWYDeHkfK5Rxn9nyBwDAiOK0mC/zzoArqbsq3YYAAJSSSm1UtiRAafeJUr8AABxTf02AfSf9Yj/cy9++45s/AABjcul+cSr0kwOVts8vPrAnpdsKAIDKibW9lWi3If2y/yK024i1uyndNgAAVNpcs3dGGbcs/uLvv/w7c83eGek2AQCgNlRqI7HKgdqustAPAABBl9NidjAjsDnhF/+mMm6ZFz8AAAGZa/bO9NcI2NWxrRPQbiPRdjXW9hZT/QAABO7S/eJUnBbzsXF3lbbPE2PfDmYIDgxl7Joy9lls3N04LeYv3S9OSV8LAAAAgDH5f6hnNXwrFNW2AAAAAElFTkSuQmCC"
```
Comment (nitrohorse) [inline _includes/sections/paste-services.html]: Don’t we already have a PNG for CryptPad we could reference?

Comment (blacklight447-ptio) [reply _includes/sections/paste-services.html]: im not sure, is it even the same logo?

Comment (nitrohorse) [reply _includes/sections/paste-services.html]: Hmm, looks like it is (under Productivity Tools, assets/img/provider/CryptPad.png).

Comment (nitrohorse) [reply _includes/sections/paste-services.html]: Just not as high quality. But I don’t think we use base64 strings for logos.

Comment (blacklight447-ptio) [reply _includes/sections/paste-services.html]: in that case i would be fine with merging, we could always change the image later if we think its not  sharp enough.

File: _includes/sections/paste-services.html [hunk 2]
```diff
@@ -7,4 +7,14 @@ <h1 id="paste" class="anchor"><a href="#paste"><i class="fas fa-link anchor-icon
...
+description="CryptPad is an open source, zero knowledge, realtime collaborative editor. Data is encrypted/decrypted in the browser using 256bit AES."
+website="https://cryptpad.fr/pad/"
+forum=""
```
Comment (Mikaela) [inline _includes/sections/paste-services.html]: This needs to be done first.

Comment (Mikaela) [inline _includes/sections/paste-services.html]:
```suggestion
forum=https://forum.privacytools.io/t/discussion-cryptpad/1270""
```

File: _includes/sections/paste-services.html (hunk 1 above)
Comment (nitrohorse) [reply _includes/sections/paste-services.html]: Hmm I think just that the string is so large, it clutters up the file a bit too much for my taste to easily work around for future changes. I can push an update later  referencing the png.

Comment (nitrohorse) [reply _includes/sections/paste-services.html]: Updated.

File: _includes/sections/paste-services.html (hunk 2 above)
Comment (nitrohorse) [reply _includes/sections/paste-services.html]: Added :+1: I tried to commit directly through the UI but apparently I "don't have permission to push to this repository" so making a new commit.

**Pull request merged**